src/components/Popup.js

```diff
@@ -3,6 +3,7 @@ export default class Popup {
 
   constructor (popupSelector) {
     this.modalElement = document.querySelector(popupSelector);
+    this.submitButton = this.modalElement.querySelector(".modal__save-button");
```

Comment:
Please, move `this.submitButton` into `PopupWithForm` and `PopupDeleteCard` because the image modal doesn't have a submit button, so you can't keep this method in `Popup`